### PR TITLE
688: added the number to the ran tests

### PR DESCRIPTION
**Pull request by lasmil**
- added the number (index) of the ran tests to differentiate between the runs
- closes: https://github.com/nearform/autocannon-ui/issues/688

### CODE REVIEW

File: packages/autocannon-ui-frontend/src/components/App.jsx
Comment (ilteoood) [inline packages/autocannon-ui-frontend/src/components/App.jsx:115]: What if you use another property to do this?

Comment (lasmil) [reply packages/autocannon-ui-frontend/src/components/App.jsx:115]: You mean to pass the index property as another one, like:
ResultSet
data = resultSet
resultIndex = index ?

Comment (ilteoood) [reply packages/autocannon-ui-frontend/src/components/App.jsx:115]: yep

Comment (lasmil) [reply packages/autocannon-ui-frontend/src/components/App.jsx]: Done

**Comment by CristiTeo** [comment]
Do we have a design for this?

**Comment by lasmil** [comment]
> Do we have a design for this?

I haven't seen it. I just added the number after the text. Should I add it somewhere else?

**Comment by lasmil** [comment]
> The number should be displayed also in compare popup

On it

File: packages/autocannon-ui-frontend/src/components/ResultSet.jsx
Comment (simoneb) [inline packages/autocannon-ui-frontend/src/components/ResultSet.jsx:56]: don't default it, this property doesn't have a sensible default

File: packages/autocannon-ui-frontend/src/components/ResultSet.jsx
Comment (simoneb) [inline packages/autocannon-ui-frontend/src/components/ResultSet.jsx:289]: make it required

**Review by simoneb — COMMENTED** [review]
We need designs for this, please check with @CristiTeo . It will be simple but designs should be aligned

File: packages/autocannon-ui-frontend/src/components/ResultSet.jsx
Comment (lasmil) [reply packages/autocannon-ui-frontend/src/components/ResultSet.jsx:289]: On it

File: packages/autocannon-ui-frontend/src/components/ResultSet.jsx
Comment (lasmil) [reply packages/autocannon-ui-frontend/src/components/ResultSet.jsx:56]: Alright, removing the default

File: packages/autocannon-ui-frontend/src/components/App.jsx
Comment (ilteoood) [inline packages/autocannon-ui-frontend/src/components/App.jsx:54]:
```suggestion
      return results.concat(resultSet).map((result, index) => {
```

Comment (lasmil) [reply packages/autocannon-ui-frontend/src/components/App.jsx]: Done. Cheers for the suggestion

**Review by ilteoood — APPROVED** [review]
LGTM

**Comment by lasmil** [comment]
@simoneb , I had a word for the designs. Travis is working on a new design (table view). I would say to get this PR merged for the time being and I will create a new task for implementing the table view. I will add as many information as I can to that task. Do you agree?

**Comment by simoneb** [comment]
We should wait until we have the design to merge this, as this PR is specifically about implementing those UI changes

**Comment by lasmil** [comment]
> We should wait until we have the design to merge this, as this PR is specifically about implementing those UI changes

Sounds like a plan. Let's do this.

**Comment by lasmil** [comment]
We got designs:
https://www.figma.com/file/f7DQ7Ev8Wk7MQKQehYphSP/Autocannon?node-id=19%3A1027

**Comment by simoneb** [comment]
@lasmil build is broken

**Comment by lasmil** [comment]
> 
I needed to retrigger it and now it is with success

File: packages/autocannon-ui-frontend/src/components/App.jsx
Comment (simoneb) [inline packages/autocannon-ui-frontend/src/components/App.jsx:37]: can we get this color from the theme instead? it's a good practice to avoid hardcoded colors and use theme colors instead

File: packages/autocannon-ui-frontend/src/components/ResultSet.jsx
Comment (simoneb) [inline packages/autocannon-ui-frontend/src/components/ResultSet.jsx:83]: 1. for spacing (e.g. margin, padding), use the spacing unit from the theme
2. for sizes, use flex sizes, avoid percentages
3. for fonts, use typography

**Review by simoneb — CHANGES_REQUESTED** [review]

File: packages/autocannon-ui-frontend/src/components/App.jsx
Comment (lasmil) [reply packages/autocannon-ui-frontend/src/components/App.jsx:37]: I agree wtih this, I will investigate

File: packages/autocannon-ui-frontend/src/components/ResultSet.jsx
Comment (lasmil) [reply packages/autocannon-ui-frontend/src/components/ResultSet.jsx:83]: On it

Comment (lasmil) [reply packages/autocannon-ui-frontend/src/components/ResultSet.jsx]: Done

File: packages/autocannon-ui-frontend/src/components/App.jsx
Comment (lasmil) [reply packages/autocannon-ui-frontend/src/components/App.jsx]: Done

**Review by simoneb — APPROVED** [review]

**Pull request merged**
